src/net.cpp

```diff
@@ -2467,28 +2528,28 @@ void StartNode(boost::thread_group &threadGroup, CScheduler &scheduler)
     // Start threads
     //
 
-    threadGroup.create_thread(boost::bind(&TraceThread<void (*)()>, "dnsseed", &ThreadAddressSeeding));
```

Comment:
The thread name functionality is useful for debugging.  We can preserve it by calling "RenameThread" defined in util.h/cpp.  But this function must be called from within the running thread.